e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/endpointFocusAndPrefix/EndpointFocusAndPrefixTest.java

```diff
@@ -0,0 +1,453 @@
+package org.evomaster.e2etests.spring.examples.endpointFocusAndPrefix;
```

Comment:
shouldn't use capital letters in package names, eg `.endpointFocusAndPrefix` should be rewritten into `.endpointfocusandprefix`. 

see `docs/for_developers.md`

Reply:
Hi Andrea. Thank you so much for your comment, I have fixed that.